src/webgl/p5.Geometry.js

```diff
@@ -174,6 +174,46 @@ p5.Geometry = class Geometry {
     this.vertexColors = [];
     return this;
   }
+
+
+  /**
+ *
+ * @method geometryToObj
+ * @for p5.Geometry
+ */
+
+  geometryToObj (fileName='model.obj'){
```

Comment:
Maybe we can call this `saveObj` so users would call it like `geom.saveObj()` instead of `geom.geometryToObj()`, which feels a tad redundant?

Reply:
`saveObj` sounds much better :)